jetty-websocket/jetty-websocket-server/src/main/java/org/eclipse/jetty/websocket/server/JettyWebSocketServletFactory.java

```diff
@@ -50,44 +49,25 @@ public WebSocketBehavior getBehavior()
         return WebSocketBehavior.SERVER;
     }
 
-    /**
-     * If true, frames are automatically fragmented to respect the maximum frame size.
-     *
-     * @return whether to automatically fragment incoming WebSocket Frames.
-     */
+    @Override
```

Comment:
Why did you remove the javadoc?
That just means the javadoc will not be present on this public class now.

Reply:
I have not removed the javadoc just moved it up to `WebSocketPolicy`, the javadoc will still be there because of the `@Override`